content/md/what-is/what-is-an-attribute.md

```diff
@@ -5,63 +5,84 @@ title: What is an **attribute**?
+- **price** attribute: a price attribute, displayed values will depend on the currencies enabled in the PIM. Learn [here](manage-your-currencies.html#enabledisable-a-currency) how to enable or disable a currency in Akeneo PIM.
+- **image** attribute: a drag and drop down zone to upload an image (extensions allowed: gif, jfif, jif, jpeg, jpg, pdf, png, psd, tif, tiff)
+- **file** attribute: a drag and drop down zone to upload a file (extensions allowed: csv, doc, docx, mp3, pdf)
+- **asset collection** attribute _(Entreprise Edition only)_: an advanced attribute type to manage several digital resources like a video, picture, pdf file... Read the following article [What is an asset?](what-is-an-asset.html) to learn more about this attribute type
+- **reference entity simple link** attribute _(Entreprise Edition only)_: allows enriching common data related to products with a rich content (text, images...), more complex than just a code and labels. To learn more about Reference entities, read our article [What is a Reference Entity?](what-about-reference-entities.html)
+- **reference entity multiple links** attribute _(Entreprise Edition only)_: the same as above but allowing you to manage multiple links
+- **reference data simple select** attribute: allows you to manage any kind of data that has its own properties, as a single-choice select. 
+- **reference data multi select** attribute: allows you to manage any kind of data that has its own properties, as a multi-choice select. 
+
+:::warning
+You can have only one attribute `identifier`, but you can have several attributes from the other types.
+:::
 
 :::info
 The only mandatory attribute is the **Identifier attribute type**. You first need to create an identifier attribute type before creating products.    
-You can only have one **identifier attribute type** in your PIM.
 :::
 
+:::tips
+Options for simple and multi select attributes can be translated in any enabled locale in the PIM.
+:::
 
 :::tips
-To store GTIN, EAN13, UPC… in your PIM, use the **text attribute type** and use a regular Expression to make sure that your codes will contain the exact number of digits required.  
-For instance, to store an EAN13 in Akeneo, which contains 13 digits, add a validation rule based on the following regular expression: `/^[0-9]{13}$/`
+To store GTIN, EAN13, UPC… in your PIM, use the **text** attribute type and use a regular expression to control the field input. 
+For instance, to store an EAN13 in Akeneo, you can easily add a validation rule based on the following regular expression: `/^[0-9]{13}$/`
 :::
 
 
-# About scopable, localisable and locale specific attributes
+# Additional attribute properties
 
-An attribute can have several specific properties: it can be localizable, locale specific and/or scopable.
+An attribute can have several specific properties: it can have a value per channel, per locale or be locale specific.
```

Comment:
Nitpicker back again: "Unique value" and "Read-only" are options that are not crystal clear for the beginner ("Usable in grid" is maybe more easily understandable)

Reply:
Good idea, we're missing the other properties (but they are described on the https://help.akeneo.com/pim/v3/articles/manage-your-attributes.html#mainContent page).
Should we add them? What do you think @KarenMayo @NolwennP and @Delphineray ?

Reply:
I've added those info